### PR TITLE
All-in-one, hindsight this would be broken up into separate commits

### PR DESCRIPTION
**Pull request by AntonPDC**
So here is the app, with both features on one branch. again I would do it differently and make a branch for each feature

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ZacharyRizer) [inline .vscode/settings.json:1]: This file should be added to your git ignore. I'll help you manage this and make sure that the git cache is removed.

File: src/App.test.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
```
Comment (ZacharyRizer) [inline src/App.test.tsx:1]: When running `npm run test a` this test is failing. We should fix the test or remove it.

File: src/App.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
-import React from 'react';
-import './App.css';
+import React from "react";
```
Comment (ZacharyRizer) [inline src/App.tsx:1]: This line is not necessary in modern React, so go ahead and remove it `import React from react`

File: src/TodoList.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
```
Comment (ZacharyRizer) [inline src/TodoList.tsx:1]: Similar thing here, change this to `import { useState } from "react"`. You no longer need to import `React` in modern react.

File: src/TodoList.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    const newTodos: item[] = todos.map((todo) => {
+      if (todo.id === id) {
+        return { ...todo, completed: !todo.completed };
+      }
+      return todo;
+    });
+    setTodos(newTodos);
+  };
+
+  // const handleToggle1 = (id: number) => {
```
Comment (ZacharyRizer) [inline src/TodoList.tsx:28]: Let's remove any unused or commented out code (unless they are actual comments).

**Review by ZacharyRizer — CHANGES_REQUESTED** [review]
See comments above. Also one more little thing, let's change what's in the `<title>` tags in `index.html` so that the app is titled `Todo App` rather than just `React App`

File: src/App.test.tsx (hunk 2 above)
Comment (AntonPDC) [reply src/App.test.tsx:1]: I think i changed the test so it should pass now

File: src/App.tsx (hunk 3 above)
Comment (AntonPDC) [reply src/App.tsx:1]: gotcha

**Review by ZacharyRizer — APPROVED** [review]
Fixes look good, approved!

**Pull request merged**